src/UKHO.ERPFacade.WebJob/Program.cs

```diff
@@ -0,0 +1,151 @@
+﻿using Azure.Extensions.AspNetCore.Configuration.Secrets;
+using Azure.Identity;
+using Azure.Security.KeyVault.Secrets;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Serilog;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using UKHO.ERPFacade.Common.Configuration;
+using UKHO.Logging.EventHubLogProvider;
+
+namespace UKHO.ERPFacade.WebJob
+{
+    [ExcludeFromCodeCoverage]
+    public static class Program
+    {
+        private static readonly InMemoryChannel s_aIChannel = new();
+        private static readonly string s_assemblyVersion = Assembly.GetExecutingAssembly().GetCustomAttributes<AssemblyFileVersionAttribute>().Single().Version;
```

Comment:
Naming standards - rename to WebJobAssemblyVersion or something similar?

Reply:
Renamed to WebJobAssemblyVersion